template/README.md

```diff
@@ -23,6 +23,8 @@ You can find the most recent version of this guide [here](https://github.com/fac
 - [Adding Custom Environment Variables](#adding-custom-environment-variables)
 - [Integrating with a Node Backend](#integrating-with-a-node-backend)
 - [Proxying API Requests in Development](#proxying-api-requests-in-development)
+- [Using Promises](#using-promises)
```

Comment:
Let’s make this one section: `Fetching AJAX Requests`. Let’s also put it before `Integrating with a Node Backend`.